README.md

```diff
@@ -17,3 +17,15 @@
 6. Parse
 7. Similar dev search
 
+Запуск и использование
+------------------------------------------
+### В терминале 
+```
+python -m  sim_dev_search.main top -r <repo_url1> -r <repo_url2> -f <out_file_path> --api-token <github_token>
```

Comment:
let's rename `main.py` to `__main__.py` - so you don't need to specify main to launch script